pkgs/data/fonts/lilex/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lilex-font";
+  version = "2.200";
+
+  src = fetchzip {
+    url = "https://github.com/mishamyrt/Lilex/releases/download/${version}/Lilex.zip";
+    sha256 = "sha256-MPQfqCMFMjcAlMos1o4bC+I+cvQYwr2TjI4Q03QeuaQ=";
+    stripRoot = false;
+  };
+
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 variable/*.ttf -t $out/share/fonts/lilex
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
Updated with requested changes in https://github.com/NixOS/nixpkgs/pull/241666/commits/6c2d2a4a520866305c9c2b0b4334e72db90864ea